pyproject.toml

```diff
@@ -44,8 +44,9 @@ doc = [
     "sphinx_rtd_theme",
 ]
 test = [
-    "blosc2>=2.5.1;python_version>='3.9'",
-    "blosc2-grok>=0.2.2;python_version>='3.9'",
+    "numpy<2 ; python_version == '3.10'",
```

Comment:
Use `numpy` v1 for Python3.9: `blosc2 wheel does not support numpy 2 but does not advertise it